woodwork/table_accessor.py

```diff
@@ -674,20 +681,35 @@ def drop(self, columns):
         if not_present:
             raise ColumnNotPresentError(not_present)
 
-        return self._get_subset_df_with_schema([col for col in self._dataframe.columns if col not in columns])
+        if inplace:
+            if dd and isinstance(self._dataframe, dd.DataFrame):
+                raise ValueError('Inplace renaming not supported for Dask')
+            if ks and isinstance(self._dataframe, ks.DataFrame):
+                raise ValueError('Inplace renaming not supported for Koalas')
+        return self._get_subset_df_with_schema([col for col in self._dataframe.columns if col not in columns], inplace=inplace)
 
-    def rename(self, columns):
+    def rename(self, columns, inplace=False):
         """Renames columns in a DataFrame, maintaining Woodwork typing information.
 
         Args:
             columns (dict[str -> str]): A dictionary mapping current column names to new column names.
+            inplace (bool): If False, return a copy. Otherwise, do operation inplace and return None.
 
         Returns:
             DataFrame: DataFrame with the specified columns renamed, maintaining Woodwork typing information.
```

Comment:
Should update to DataFrame or None here and in Drop

Reply:
Fixed!